Kosystem/Services/IRoomRepository.cs

```diff
@@ -0,0 +1,22 @@
+using System.Collections.Generic;
+using Kosystem.Shared;
+
+namespace Kosystem.Services
+{
+    public interface IRoomRepository
+    {
+        RoomModel CreateRoom(NewRoomModel newRoom);
+
+        bool DeleteRoom(int roomId);
+
+        RoomModel? UpdateRoom(UpdateRoomModel updateRoom);
```

Comment:
Codacy found an issue: [Rename the parameter 'updateRoom' so that it does not duplicate the method name.](https://app.codacy.com/gh/HelloWorldSweden/Kosystem/pullRequest?prid=6010716)